yarp.gemspec

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name = "yarp"
+  spec.version = "0.4.0"
+  spec.authors = ["Shopify"]
+  spec.email = ["ruby@shopify.com"]
```

Comment:
This is funny, didn't know we had this email address!

Reply:
To all of you https://groups.google.com/a/shopify.com/g/ruby/members